### ISSUE TITLE
After clicking "Trigger backup", the application crashes.

### ISSUE THREAD

**Issue by Vstory** [beemdevelopment/Aegis#468]
###### Info

* __Version__: 1.2-beta5
* __Source__: Google Play
* __Vault encrypted:__ Yes (with biometric unlock)  Yes
* __Device:__ Xiaomi Note 3
* __Android version and ROM:__ Android 9 ,MIUI 12 20.6.11

###### Steps to reproduce
Open "aegis", click "trigger backup", the application crashes.

###### What do you expect to happen?
backup database

...

###### What happens instead?
After clicking "Trigger backup", the application crashes.

...

###### Log

```
java.lang.IllegalArgumentException: Invalid URI: content://com.android.externalstorage.documents/document/home%3Aaegis-export.json-20200526-222451.json
	at android.provider.DocumentsContract.getTreeDocumentId(DocumentsContract.java:1014)
	at com.beemdevelopment.aegis.vault.VaultManager.backup(VaultManager.java:8)
	at com.beemdevelopment.aegis.ui.PreferencesFragment.lambda$onCreatePreferences$9$PreferencesFragment(PreferencesFragment.java:2)
	at com.beemdevelopment.aegis.ui.-$$Lambda$PreferencesFragment$D6LYjVCYWE6cirBWoFcMBCg545s.onPreferenceClick(Unknown Source:2)
	at androidx.preference.Preference.performClick(Preference.java:4)
	at androidx.preference.Preference$1.onClick(Preference.java:1)
	at android.view.View.performClick(View.java:6608)
	at android.view.View.performClickInternal(View.java:6585)
	at android.view.View.access$3100(View.java:785)
	at android.view.View$PerformClick.run(View.java:25921)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:201)
	at android.app.ActivityThread.main(ActivityThread.java:6821)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)
```

**Comment by alexbakker** [comment]
Thanks for your report. What happens if you disable and then reenable backups? We had a bug in one of our beta's where backups would inadvertently and incorrectly be configured when exporting the vault.

**Comment by Vstory** [comment]
> Thanks for your report. What happens if you disable and then reenable backups? We had a bug in one of our beta's where backups would inadvertently and incorrectly be configured when exporting the vault.


Each backup must follow the operation you said before it can be successfully backed up.Otherwise, click directly to trigger the backup, and it will still crash.

**Comment by alexbakker** [comment]
Could you rephrase that? I'm not sure what you mean. Do you mean that after enabling backups, "trigger backup" only works once and crashes the app after a second try?

**Comment by Vstory** [comment]
> Could you rephrase that? I'm not sure what you mean. Do you mean that after enabling backups, "trigger backup" only works once and crashes the app after a second try?


Version 1.2 has solved the problem.
